dnf-plugins-extras.spec

```diff
@@ -257,6 +257,47 @@ Obsoletes:      dnf-plugins-extras-snapper <= 0.0.4-2
 %description -n python3-dnf-plugins-extras-snapper
 Snapper Plugin for DNF, Python 3 version. Creates snapshot every transaction.
 
+%package -n python2-%{name}-system-upgrade
+Summary:        System Upgrade Plugin for DNF
+Requires:       python-%{name}-common = %{?epoch:%{?epoch}:}%{version}-%{release}
+Requires:       python2-systemd
+%{?python_provide:%python_provide python2-%{name}-system-upgrade}
+
+Obsoletes:	fedup < 0.9.4
+Obsoletes:	dnf-plugin-system-upgrade < 0.10
+Obsoletes:	python2-dnf-plugin-system-upgrade < 0.10
+
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  systemd
+BuildRequires:  python2-systemd
+%{?system_requires}
```

Comment:
What are you talking about, they are there. It's above the BuildRequires. Also, I think you mean "python2-systemd" here.